docs/Getting_Involved/Testing.rst

```diff
@@ -9,12 +9,26 @@ Executing our Tests
 *coala* has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
-To run tests, you first need to install some dependencies. This can be
-done by executing:
+To run tests, you first need to install some dependencies.
+This can be done by following these steps:
+
+If you have not already, You can clone the
+`repository <https://github.com/coala-analyzer/coala>`_ (or a fork of
+it) by running:
+
+::
+
+    $ git clone https://github.com/coala-analyzer/coala.git
+
+Then navigate to the directory where coala is located.
```

Comment:
As earlier, these are steps, you should imperative it: ``Navigate to the..`` - no ``then``